src/Microsoft.Framework.Runtime.Interfaces/Microsoft.Framework.Runtime.Interfaces.xproj

```diff
@@ -10,8 +10,14 @@
     <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)'=='' ">..\..\artifacts\obj\$(MSBuildProjectName)</BaseIntermediateOutputPath>
     <OutputPath Condition="'$(OutputPath)'=='' ">..\..\artifacts\bin\$(MSBuildProjectName)\</OutputPath>
   </PropertyGroup>
+  <PropertyGroup Label="Configuration" Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
The RootNamespace was still "M.F.R.Interfaces", meaning it's really easy for new types to end up with that namespace. This sets it to "M.F.Runtime" (correct me if I'm wrong, but I thought M.F.R.Interfaces was an assembly only, not a namespace). Just a VS hint.